src/Installer/redist-installer/targets/GenerateRuntimeAnalyzers.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
this file can be deleted, it was removed in https://github.com/dotnet/sdk/pull/47397